config/tari_config_sample.toml

```diff
@@ -91,10 +91,6 @@
 # peer_seeds = ["public_key1::address1", "public_key2::address2",... ]
 peer_seeds = []
 
-# Configure the number of threads to spawn for long-running tasks, like block and transaction validation. A good choice
-# for this value is somewhere between n/2 and n - 1, where n is the number of cores on your machine.
-#blocking_threads = 4
-
 # Configure the number of threads for the overall node work-stealing scheduler. A good value here is between 3 and
```

Comment:
This shouldn't be nuked

Reply:
Fixed - not sure how that slipped in again